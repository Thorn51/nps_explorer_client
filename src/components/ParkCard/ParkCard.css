:root {
  --darkGunmetal: #122932;
  --blackCoral: #576066;
  --queenBlue: #386fa4;
  --middleBlue: #80ced7;
  --mintCream: #f7fff7;
  --platinum: #e0e2e3;
}

/* Homepage park cards */

.featured_parks {
  display: flex;
  flex-direction: row;
  justify-content: space-around;
  flex-wrap: wrap;
  padding: 15px;
}

.park_card {
  display: block;
  height: 300px;
  width: 300px;
  margin-top: 10px;
  background: white;
  border: var(--darkGunmetal) solid 2px;
  box-shadow: 2px 2px 2px black;
}

.pc_designation {
  text-align: center;
  color: white;
  padding: 5px;
  background-color: var(--darkGunmetal);
}

.pc_name {
  text-align: center;
  padding: 10px;
}

.pc_description {
  text-align: left;
  padding: 5px;
}
