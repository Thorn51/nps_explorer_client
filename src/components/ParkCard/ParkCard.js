import React from "react";
import "./ParkCard.css";

export default function ParkCard() {
  return <div></div>;
}
