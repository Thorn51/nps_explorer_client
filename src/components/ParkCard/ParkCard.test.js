import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import ParkCard from "./ParkCard";
import { BrowserRouter } from "react-router-dom";

describe("<ParkCard />", () => {
  it("Renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <ParkCard />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it("Renders the UI as expected", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <ParkCard />
        </BrowserRouter>
      )
      .toJSON();
    expect("tree").toMatchSnapshot();
  });
});
