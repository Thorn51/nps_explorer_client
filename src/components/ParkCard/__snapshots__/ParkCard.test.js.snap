// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ParkCard /> Renders the UI as expected 1`] = `"tree"`;
